src/main/java/org/example/Main.java

```diff
@@ -1,5 +1,49 @@
-package org.example;public class Main {
+package org.example;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/*
+1. Создать программу для сохранения заказа на покупку товара в файл.
+Данные о заказе (наименование товара, количество, общая стоимость)
+вводятся пользователем с клавиатуры.
+Сделать логироване действий пользователя в файл.
+Предусмотреть различные уровни логирования: общая информация,
+предупреждающее сообщения, сообщения об ошибках.
+
+2. Для предыдущего задания создать unit-тесты. Предусмотреть покрытие
+кода тестами минимум на 50%.
+ */
+public class Main {
+
+    private static Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
-    }
-}
\ No newline at end of file
+
+            Scanner in = new Scanner(System.in);
+
+            try(FileWriter writer = new FileWriter("Orders.txt"))
+            {
+                logger.info("Вводим данные о наименовании товара");
+                System.out.println("Введите наименование товара:");
+                writer.write("Наименовани товара: "+in.nextLine()+"\n");
```

Comment:
Why did you decide to process input params without speciall class, eg. Order?
It seems, if you have any exception in the middle of the code, yje result file will contain incorrect information